domains/william2sober.json

```
@@ -0,0 +1,14 @@
+{
+  "description": "Documentation website for is-a.dev",
+  "repo": "https://github.com/william2sober/is-a-dev",
+  "owner": {
+    "username": "william2sober.is-a-dev",
```

does not match github username
```suggestion
    "username": "william2sober",
```